activemodel/lib/active_model/validations/callbacks.rb

```diff
@@ -23,6 +23,7 @@ module Callbacks
       included do
         include ActiveSupport::Callbacks
         define_callbacks :validation,
+                         terminator: deprecated_false_terminator(ActiveModel::Validations),
```

Comment:
Indeed. It looks simpler to say `Active Record and Active Model` instead of using an argument.

All in favor of changing it back and improving the message?